### DISCUSSION TITLE
Console support for stride engine

### DISCUSSION THREAD

**Discussion by ClockWritesDev** [stride3d/stride#1319] [Q&A]
Is there any way to add a new sdk on stride to Port the games to console ?

Please make a tutorial how to add new platforms to stride.

**Comment by NicusorN5** [comment]
I'm not sure, but for PlayStation, the code needs to be converted to C++ and implement some API calls? I'm not quite sure about that.

I think you could run UWP applications on Xbox :https://docs.microsoft.com/en-us/windows/uwp/xbox-apps/

**Comment by Notexe** [comment]
There is this https://www.mono-project.com/docs/about-mono/supported-platforms/playstation4/ but I'm not sure how different mono is compared to .net.
